eth/main.cpp

```diff
@@ -1062,8 +1062,8 @@ int main(int argc, char** argv)
 
     web3.setIdealPeerCount(peers);
     web3.setPeerStretch(peerStretch);
-//  std::shared_ptr<eth::BasicGasPricer> gasPricer = make_shared<eth::BasicGasPricer>(u256(double(ether / 1000) / etherPrice), u256(blockFees * 1000));
-    std::shared_ptr<eth::TrivialGasPricer> gasPricer = make_shared<eth::TrivialGasPricer>(askPrice, bidPrice);
+    std::shared_ptr<eth::TrivialGasPricer> gasPricer =
+        make_shared<eth::TrivialGasPricer>(askPrice, testingMode ? 0 : bidPrice);
```

Comment:
Why big price must be 0 in testing mode?

Reply:
checking it now. looks like it does not affect the tests.

Reply:
It should not affect test transactions unless the transactions does not have gas price specified. In other words this is tx gas price in case user does not specify any.